event/src/unvalidated.rs

```diff
@@ -11,13 +11,15 @@ use crate::bytes::Bytes;
 /// Ceramic Event
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(untagged)]
-pub enum Event {
+pub enum Event<D> {
     /// Time event in a stream
     // NOTE: TimeEvent has several CIDs so its a relatively large struct (~312 bytes according to
     // the compiler). Therefore we box it here to keep the Event enum small.
     Time(Box<TimeEvent>),
     /// Signed event in a stream
     Signed(JWS),
+    /// Unsigned event in a stream
+    Unsigned(InitPayload<D>),
```

Comment:
not important or worth changing in this PR, but could the payload *ever* be anything other than Ipld?  Is there a reason to keep this generic?

Reply:
The idea is if you know what it is say in specific application code you can directly deserialize into that value instead of first deserializing into IPLD and then reconstructing the application specific data from the IPLD data. This was how the Event API was setup initially for something that @dbcfd was using. Not sure what specific use case he needed it for but that is the idea.